frontend/src/components/about/Value.js

```diff
@@ -1,7 +1,102 @@
 import React from 'react'
+import Box from '@mui/material/Box';
+import { ThemeProvider } from '@emotion/react';
+import { Theme } from '../../config/Theme';
+import { Typography } from '@mui/material';
 
+
+const values = [{
+    title: "1. Mission First", body: "We as Blueprint members ultimately unify under one goal - achieving our mission. We place the interests of the people and partners we serve above our own. Our primary measure of success is the amount of positive impact we create through our work."
+    , image: "/images/mountain.png"
+},
+{
+    title: "2. Perpetual Growth", body: "We as Blueprint members value the academic, social, and personal growth of our peers. We constantly seek to perpetuate the cycle of learning and teaching, for our benefit and for others.We strive to offer a helping hand in times of need and push each other to succeed."
+    , image: "/images/growth.png"
+},
+{
+    title: "3. Cherish Each Other", body: "We as Blueprint members seek to cherish our time spent together. We develop meaningful relationships that extend well beyond the scope of the organization. We value each other as individuals and appreciate our differences."
+    , image: "/images/love.png"
+},
+{
+    title: "4. Always Innovate", body: "We as Blueprint members value the academic, social, and personal growth of our peers. We constantly seek to perpetuate the cycle of learning and teaching, for our benefit and for others. We strive to offer a helping hand in times of need and push each other to succeed."
+    , image: "/images/rocket.png"
+},
+{
+    title: "5. Be Humble", body: "We as Blueprint members strive to remain humble, accept our imperfections, and be receptive to feedback. We approach challenges with an open mind and remember that anyone can pursue social good, not just Blueprint."
+    , image: "/images/support.png"
+}];
+
+const display = [];
+
+
+for (var i = 1; i < (values.length + 1); i++) {
+    //Odd
+    if (i % 2 !== 0) {
+        display.push((<Box sx={{
```

Comment:
I think the content inside the <Box> should be moved to a component on its own e.g., LeftValue or OddValue